### PR TITLE
[ALLUXIO-1796] Made NavBar Responsive in smaller window

### PR DESCRIPTION
**Pull request by xxhuang**
Changed bootstrap responsive breakpoints in bootstrap-responsive.min.css and and made the header navigation bar collapse in smaller window

JIRA: https://alluxio.atlassian.net/browse/ALLUXIO-1796

### CODE REVIEW

**Comment by yupeng9** [comment]
okay to test

**Comment by calvinjia** [comment]
Jenkins, ok to test.

File: core/server/src/main/webapp/header.jsp [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+        <% } %>
+      </ul>
+    </div>
   </div>
 </div>
 <script>
   var url = window.location.pathname;
   $("#"+url.substring(1)+"-li").addClass("active");
 </script>
+<link rel="stylesheet" type="text/css" href="css/bootstrap-responsive.min.css">
```
Comment (uronce-cc) [inline core/server/src/main/webapp/header.jsp]: Add missing new line

**Comment by uronce-cc** [comment]
screen shot 2016-03-26 at 10 49 25 am

**Comment by uronce-cc** [comment]
@xxhuang Thanks for making the UI responsive! I left a minor comment. 

Otherwise, LGTM.

File: core/server/src/main/webapp/header.jsp (hunk 1 above)
Comment (xxhuang) [inline core/server/src/main/webapp/header.jsp]: Thanks for comment. I am thinking maybe we put the link in the webapp/general.jsp would be better?

Comment (uronce-cc) [inline core/server/src/main/webapp/header.jsp]: @xxhuang yes, for other css imports, they are put in the header blocks. But if it is put in webapp/general.jsp, I think it also needs to be put in [webapp/configuration.jsp](https://github.com/Alluxio/alluxio/blob/master/core/server/src/main/webapp/configuration.jsp#L18) etc.

Comment (uronce-cc) [inline core/server/src/main/webapp/header.jsp]: I think we could use a similar mechanism like [jsp include](https://github.com/Alluxio/alluxio/blob/master/core/server/src/main/webapp/configuration.jsp#L24) to aggregate all header blocks into one file.

Comment (xxhuang) [inline core/server/src/main/webapp/header.jsp]: I think this is a good idea. Let me do it

Comment (xxhuang) [inline core/server/src/main/webapp/header.jsp]: @uronce-cc I think this is a good idea. Let me do it

Comment (uronce-cc) [inline core/server/src/main/webapp/header.jsp]: @xxhuang Sounds good! Looking forward to the refactored codes!

**Comment by xxhuang** [comment]
@uronce-cc PTAL

File: core/server/src/main/webapp/header-links.jsp [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<link href="css/bootstrap.min.css" rel="stylesheet" media="screen">
+<link href="css/custom.min.css" rel="stylesheet">
+<link href="img/favicon.ico" rel="shortcut icon">
+<link href="css/bootstrap-responsive.min.css" rel="stylesheet">
```
Comment (uronce-cc) [inline core/server/src/main/webapp/header-links.jsp]: put all css links together

**Comment by uronce-cc** [comment]
@xxhuang looks good! I think the same improvement of aggregating all headers together can be applied to the jps files under webapp/worker.

**Comment by xxhuang** [comment]
Oops I missed the webapp/worker folder! Thanks for letting me know @uronce-cc ! PTAL!

**Comment by uronce-cc** [comment]
@xxhuang Thanks for improving the frontend!

LGTM

File: core/server/src/main/webapp/header.jsp [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
+        <% } %>
+      </ul>
+    </div>
   </div>
 </div>
 <script>
   var url = window.location.pathname;
   $("#"+url.substring(1)+"-li").addClass("active");
-</script>
+</script>
```
Comment (yupeng9) [inline core/server/src/main/webapp/header.jsp]: add a new line

**Comment by yupeng9** [comment]
Overall looks good to me. I just left one minor comment.

**Comment by yupeng9** [comment]
Also, could you update the screenshot? Thanks.

**Comment by xxhuang** [comment]
**Nav Bar in Medium Window**

screen shot 2016-03-30 at 12 22 11 am

**Nav Bar Menu**

screen shot 2016-03-30 at 12 22 23 am

**Nav bar in Small Window**

screen shot 2016-03-30 at 12 22 38 am

@yupeng9

**Comment by yupeng9** [comment]
This looks awesome. Thanks!

**Pull request merged**